build.gradle

```diff
@@ -19,12 +19,12 @@ plugins {
 
 version = '3.3.2-SNAPSHOT'
 ext {
-    compileSdkVersion = 28
+    compileSdkVersion = 30
     minSdkVersion = 21
-    targetSdkVersion = 28
+    targetSdkVersion = 30
     versionCode = 11
     versionName = version
-    braintreeVersion = '3.7.0'
+    braintreeVersion = '3.14.1'
```

Comment:
Does this need to be mentioned in the CHANGELOG? Is this the equivalent of saying that this version of braintree-android-google-payment requires braintree 3.14.1 or higher?

Reply:
I'm not 100% sure since the dependency is marked as `compileOnly`, but we do specify this as the latest braintree is used in the [docs](https://developers.braintreepayments.com/guides/google-pay/client-side/android/v3#add-braintree-googlepayment-component).

Reply:
I know for `implementation` style dependencies gradle will pick the highest version. `compileOnly` is a bit different. It'll require merchants to add the braintree library as a peer dependency.

Reply:
We could leave it? `3.7.0` could probably be considered an implicit min version for the braintree lib.

Reply:
Gotcha - thanks for clarifying